### PR TITLE
RATIS-1451. GroupManagementImpl should use sendRequestWithRetry.

### PR DESCRIPTION
**Pull request by szetszwo**
See https://issues.apache.org/jira/browse/RATIS-1451

### CODE REVIEW

File: ratis-client/src/main/java/org/apache/ratis/client/impl/BlockingImpl.java [hunk 1]
```diff
@@ -99,7 +100,7 @@ public RaftClientRequest newRequestImpl() {
           return client.handleReply(request, reply);
         }
       } catch (GroupMismatchException | StateMachineException | TransferLeadershipException |
-          LeaderSteppingDownException e) {
+          LeaderSteppingDownException | AlreadyExistsException e) {
```
Comment (adoroszlai) [inline ratis-client/src/main/java/org/apache/ratis/client/impl/BlockingImpl.java:103]: Should we also avoid retry for `AlreadyClosedException`?

**Review by adoroszlai — APPROVED** [review]
Thanks @szetszwo for the fix, it looks good to me, just have one question.

File: ratis-client/src/main/java/org/apache/ratis/client/impl/BlockingImpl.java (hunk 1 above)
Comment (szetszwo) [reply ratis-client/src/main/java/org/apache/ratis/client/impl/BlockingImpl.java]: Sure.  Just have added it.

**Comment by adoroszlai** [comment]
Thanks @szetszwo for updating the patch.  Finally we have a green build.

**Pull request merged**